bluetooth/src/commonMain/kotlin/ContextualLogger.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ Copyright 2025 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.bluetooth
+
+import com.splendo.kaluga.logging.LogLevel
+import com.splendo.kaluga.logging.Logger
+
+/**
+ * A logger appending contextual data to each message
+ */
+class ContextualLogger(private val logger: Logger, private val tag: String, private val context: Map<String, Any?> = LinkedHashMap()) : Logger {
```

Comment:
This should move to the Kaluga logging library

Reply:
moved